components/Orders/ProposalOpenOrderRow.tsx

```diff
@@ -107,6 +111,22 @@ export function ProposalOpenOrderRow({ order }: { order: OpenOrdersAccountWithKe
     try {
       setIsEditing(true);
       await sender.send(txs);
+      // TODO, once edit working as expected, enable this
```

Comment:
@R-K-H I have had a weird experience with this where it reliably does not edit the order, but rather just closes the order and then allows me to settled in the unsettled tab. I assume this isn't desired functionality? Have you seen this?

Reply:
haha did it really work during those weekends? I'm pretty sus at this point. But okay I'll take your word for it.